nni/algorithms/hpo/dngo_tuner/dngo_tuner.py

```diff
@@ -0,0 +1,117 @@
+import random
```

Comment:
I think a `__init__.py` is needed to make this importable.

Reply:
fixed it.